### PR TITLE
[DCSFUS-2] Developed a product Page using HTML

### PR DESCRIPTION
**Pull request by Saypauloh**
This is my second pull request
In this PR, i have added my second task of creating an ecommerce homepage.

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
The homepage.html file should be in the HTML folder. Remove the dufuna folder and rename the homepage folder to HTML.

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Also, your images should be in an images folder. You don't have to create several folders.

File: submissions/paulOlubunmi/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    </nav>
+
+    <section>
+
+        <h2><b>Phones & Accessories</b></h2>
+        <p>
+            <h4> Shopping has never been this easy!</h4>
+            <image src="image/pa1.jpg "></image>
+            <image src="image/pa2.jpg "></image>
+           <!-- <image src="image/pa3.jpg "></image>-->
```
Comment (Ireti203) [inline submissions/paulOlubunmi/HTML/index.html:22]: Remove commented out code.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Your product page ought to have only 2 images, but I can see that you have more than 2. Kindly update that.

Always ensure you follow the instructions on the task.

**Comment by Saypauloh** [comment]
Done ma

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**